fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/TransactionObject.java

```diff
@@ -192,6 +188,54 @@ public long getMiniLaoPerReceiver(PublicKey receiver) {
     return miniLao;
   }
 
+  /**
+   * Total MiniLao per public key of a List of Transaction
+   *
+   * @param transaction List<TransactionObject>
+   * @param receiver Public Key
+   * @return long amount per user
+   */
+  public static long getMiniLaoPerReceiverSetTransaction(
+      List<TransactionObject> transaction, PublicKey receiver) {
+    Iterator<TransactionObject> transactionIte = transaction.iterator();
+    long totalAmount = 0;
+    while (transactionIte.hasNext()) {
+      long current = transactionIte.next().getMiniLaoPerReceiver(receiver);
+      totalAmount += current;
+    }
+    return totalAmount;
```

Comment:
Random fact, here, the use of streams (functional-like programming) is quite useful :

```suggestion
    return transaction.stream()
            .mapToLong(obj -> obj.getMiniLaoPerReceiver(receiver))
            .sum();
```